core/src/main/scala/cromwell/core/simpleton/WdlValueBuilder.scala

```diff
@@ -0,0 +1,145 @@
+package cromwell.core.simpleton
+
+import wdl4s.TaskOutput
+import wdl4s.types.{WdlArrayType, WdlMapType, WdlPrimitiveType, WdlType}
+import wdl4s.values.{WdlArray, WdlMap, WdlValue}
+
+import scala.language.postfixOps
+import cromwell.core.simpleton.WdlValueSimpleton._
+
+
+/** Builds arbitrary WdlValues from WdlValueSimpletons. */
+object WdlValueBuilder {
+
+  private val SimpletonPattern = """
```

Comment:
I appreciate the effort but I'm thinking that I'd read this easier if it was just the regex w/ a comment explaining what was going on. YMMV and all that, if others disagree so be it